### ISSUE TITLE
useAnimatedScrollHandler with refreshControl not working

### ISSUE THREAD

**Issue by SumitSharmaJii** [software-mansion/react-native-reanimated#5972]
### Description

useAnimatedScrollHandler  is not working on Android with RefreshControl similar to https://github.com/software-mansion/react-native-reanimated/issues/3937

### Steps to reproduce

```
<Animated.ScrollView 
    onScroll={scrollHandler}
    refreshControl={<RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
}> {children}  </Animated.ScrollView>
```

### Snack or a link to a repository

https://snack.expo.dev/OAXfUmXwP1DilS_uErLED

### Reanimated version

3.10.0

### React Native version

0.74.1

### Platforms

Android

### JavaScript runtime

None

### Workflow

None

### Architecture

None

### Build type

None

### Device

None

### Acknowledgements

Yes

**Comment by bogdanovic91** [comment]
same issue here

**Comment by Ravnravn** [comment]
This [patch](https://github.com/software-mansion/react-native-reanimated/pull/5960#issuecomment-2098548412) fixes the issue on 3.11.0

**Comment by szydlovsky** [comment]
Hello everyone, the patch @Ravnravn mentioned is already included in `3.12 version` - please upgrade Reanimated and it should be gone 👍
